SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,152 @@
+# Next steps: Set up the SpatialOS GDK Starter template
+
+Before setting up the SpatialOS GDK Starter template, you need to have followed:
+
+* [Getting started: 1 - Dependencies]({{urlRoot}}/content/get-started/dependencies)
+* [Getting started: 2 - Get and build the SpatialOS Unreal Engine Fork]({{urlRoot}}/content/get-started/tutorial).
+
+If you haven't already, follow our [Multiserver Shooter tutorial]({{urlRoot}}/content/get-started/dependencies) to try out the SpatialOS for Unreal GDK development experience by adding a new feature to a project and testing it across two servers.
+
+If you are ready to start developing your own game with the GDK, follow the steps below. 
+
+### Terms used on this page
+
+* `<Game>` - The directory that contains your project’s .uproject file and Source folder.
+* `<ProjectRoot>` - The directory that contains your `<Game>` directory.
+* `<YourProject>` - The name of your project and .uproject file (for example, `\<Game>\YourProject.uproject`).
+
+### Create a new project using the starter template
+
+After [building the Unreal Engine fork]({{urlRoot}}/content/get-started/build-unreal-fork), in **File Explorer**, navigate to `UnrealEngine\Engine\Binaries\Win64`and double-click UE4Editor.exe to open the Unreal Editor. 
+
+In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, click the New Project tab and select the C++ tab, then select the **SpatialOS GDK Starter** template. Choose a suitable directory and name for your project, then click **Create Project**. 
```

Comment:
```suggestion
1. In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, select the **New Project** tab and then the **C++ tab**. 
2. In this tab, select **SpatialOS GDK Starter** template. Enter a directory name project name of your choice <where to do do this?>, and select **Create Project**. 

Have suggested the MMoS standard 'select' and assuming the text in bold is on the UI?
Have suggested using two steps to break this set of instructions up.
Assume our user makes up their directory and file name? - we should be explicit with this, I think.



```

Reply:
```suggestion
1. In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, select the **New Project** tab and then the **C++ tab**. 
2. In this tab, select **SpatialOS GDK Starter** template. Enter a directory name and project name of your choice, and select **Create Project**.
```
I think the screenshot highlights the user-defined sections well enough and given that we're aiming the docs at user's familiar with Unreal we shouldn't have to hand-hold them quite that much with the basics. 

We could have steps that walk users through choosing a directory and name but I think that might make this section a bit unwieldy. 

Good catch on the MMoS standard for select. I thought I caught them all! I'll go through and fix any "clicks".

Reply:
>I think the screenshot highlights the user-defined sections well enough and given that we're >aiming the docs at users familiar with Unreal we shouldn't have to hand-hold them quite that > much with the basics.
I can't see it. Have you got a screen grab or anything on testing or staging to see the context?
Thinking that we have a standard of **bold** to make it clear when we are referring to text in the UI?
Also thinking that we direct non-Unreal users who might BYO engine to the GDK for Unreal as a way to get to use SpatialOS. So we want to make these early steps as pain free.

>We could have steps that walk users through choosing a directory and name but I think that might make this section a bit unwieldy.

Thinking more `In the **UI title whatever it is** field, enter a directory name and project name of your choice...`

Also wondering if it should be a new directory and a new project name? This isn't clear. As we are using the Starter template, it might seem that I have to enter an existing one?

Reply:
Also, just realised you create a project name and Unreal creates the directory to contain it with a matching name, so saying "enter a project name of your choice" is probably clearer, especially as there is an explanation below.

Reply:
```suggestion
1. In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, select the **New Project** tab and then the **C++ tab**. 
2. In this tab, select **SpatialOS GDK Starter** template. Enter a project name of your choice and select a directory for your project to be stored in, this directory will become your <ProjectRoot>. 
3. Select **Create Project**.
```

?

Reply:
```Suggestion
1. In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, select the **New Project** tab and then the **C++ tab**. 
2. In this tab, select **SpatialOS GDK Starter**. 
3. In the **Folder** field, choose a suitable directory for your project.
4. In the **Name** field, enter a project name of your choice.
5. Select **Create Project**.
```

Gone with this.